README.md

```
@@ -662,6 +699,96 @@ as configuration. The dispatcher runs archive compaction as part of its
+
+Pgbus ships two log formatters inspired by Sidekiq's `Logger::Formatters`:
+
+```ruby
+Pgbus.configure do |c|
+  c.log_format = :json   # or :text (default)
+end
+```
+
+**Text format** (default):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)